### ISSUE TITLE
[Accessibility] Accessibility Magic Tap should launch/hide Reader View

### ISSUE THREAD

**Issue by mozillamobilebot** [mozilla-mobile/firefox-ios#4982]
From https://bugzilla.mozilla.org/show_bug.cgi?id=1160786

User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_10_3) AppleWebKit/600.5.17 (KHTML, like Gecko) Version/8.0.5 Safari/600.5.17

Steps to reproduce:

When on a webpage where reader mode is available, VoiceOver user, while having the webpage displayed, should be able to perform a Magic Tap from anywhere (i.e. right after opening the webpage) to launch Reader View, and also hide it.

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FXIOS-2108)

**Comment by SimonBasca** [comment]
From my understanding, Magic Trap is a 2 finger tap gesture that allows the user to perform a certain action. This seems more like a feature request.

**Comment by data-sync-user** [comment]
➤ Andrei Bodea commented:

Verified as fixed on v123 (38283) with iPhone 15 Pro (17.1.2).
